packages/playwright-test/types/test.d.ts

```diff
@@ -16,14 +16,18 @@
  */
 
 import type { APIRequestContext, Browser, BrowserContext, BrowserContextOptions, Page, LaunchOptions, ViewportSize, Geolocation, HTTPCredentials, Locator, APIResponse, PageScreenshotOptions } from 'playwright-core';
+import type { TestCase } from '@playwright/test/reporter';
+
 export * from 'playwright-core';
 
+export type junitClassNameTemplate = (test: TestCase) => string;
```

Comment:
`TestCase` parameter is not sufficient to get e.g. a relative path of the test file, without it you won't be able to implement default classname format in the user code. Having the config as another parameter would help with that.